docs/docs/CPG/specs/overlays.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Overlay Graph"
+linkTitle: "Overlay Graph"
+no_list: true
+weight: 1
+date: 2025-01-10
+description: >
+    Overlay Graph
+---
+
+# Overlay Graph
+
+The CPG represents the code of a program as a graph of nodes $N_{CPG}$
+and edges $E$.
+
+Our basic version of the CPG only considers nodes that are part
+of the CPG's immediate representation of the program's AST (we denote
+these nodes as $N_{AST} \subseteq N_{CPG}$).
```

Comment:
For the latex syntax to work we would need to fetch an additional javascript pkg from unpkg.com or similar. Do we want/need that? See https://squidfunk.github.io/mkdocs-material/reference/math/#mathjax

Reply:
I leave this decision to @konradweiss

Reply:
I added a local version of MathJax